stations.csv

```diff
@@ -8611,7 +8611,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 10487;Jenbach;jenbach;8101176;;47.3884510000;11.7784570000;;AT;Europe/Vienna;f;f;f;t;f;f;ATJEN;;f;;f;8100102;t;;f;;f;;f;;f;;f;8100102;t;;f;8101176;f;;;;;f;;f;;f;;f;ATJEN;t;;f;f;;;;;;;;;;;;;;;;Енбах;;;延巴赫;urn:trainline:public:nloc:csv10487;
 10488;Smolensk;smolensk;2017010;;54.777369;32.027504;;RU;Europe/Moscow;f;f;f;f;f;f;RUSAA;;f;;f;2000004;f;;f;;f;;f;;f;;f;2000004;t;;f;;f;;;;;f;;f;;f;;f;RUSMO;f;;f;f;;;;;;;;;;;Szmolenszk;スモレンスク;스몰렌스크;;;;;;斯摩棱斯克;urn:trainline:public:nloc:csv10488;
 10489;Minsk;minsk;2114020;;53.868712;27.651025;27861;BY;Europe/Minsk;t;f;f;t;f;f;BYMIP;;f;;f;2196004;f;;f;BYMSQ;t;8028;t;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;Minszk;ミンスク;민스크;;;Минск;;;明斯克;urn:trainline:public:nloc:csv10489;
-10490;Brest-Centralnyi;brest-centralnyi;2113000;;52.100529;23.680661;;BY;Europe/Minsk;f;f;f;f;t;f;BYBRC;;f;;f;2100001;f;;f;BYBQT;t;23218;t;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;BYBRC;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv10490;
+10490;Brest-Centralnyi;brest-centralnyi;2113000;;52.100529;23.680661;;BY;Europe/Minsk;f;f;f;f;t;f;BYBRC;;f;;f;2100001;f;;f;;f;23218;t;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;BYBRC;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv10490;
```

Comment:
Why was this station removed?

Reply:
It's not present in the distribution locations anymore